docs/sdk/authentication/auth-sig.md

```diff
@@ -39,9 +39,14 @@ import { checkAndSignAuthMessage } from '@lit-protocol/lit-node-client';
 
 const authSig = await checkAndSignAuthMessage({
   chain: "ethereum",
+  nonce,
 });
 ```
 
+:::note
+Be sure to use the latest blockhash from the `litNodeClient` as the nonce. You can get it from the `litNodeClient.getLatestBlockhash()`.
```

Comment:
can we also show how to do this if not using a web extension wallet?

Reply:
I already have. Check the "Obtaining an AuthSig on the server-side" section below

Reply:
I see that now, looks good, and approved.